gradle/scala.gradle

```diff
@@ -49,12 +49,18 @@ test {
 sourceSets {
     main {
         scala {
-            srcDir "src/main/scala_$scalaBaseVersion"
+            srcDirs += ["src/main/scala_$scalaBaseVersion", "src/main/java"]
+        }
+        java {
+            srcDirs = []
         }
     }
```

Comment:
No, putting it in separate sets (java and scala) won't work because the Java classes depend on Scala classes and they won't see each other.

Hm I tried `srcDirs ["src/main/scala_$scalaBaseVersion", "src/main/java"]` but it didn't work (which was weird, the API has a srcDirs() method but I got some gradle exceptions) and `srcDir` doesn't take sets (only single dirs). Do you know what should be the exact syntax?

I did notice I was adding `main` in the `test` sources, though :-)